examples/src/main/java/io/dapr/examples/bindings/http/OutputBindingExample.java

```diff
@@ -28,6 +28,8 @@ public MyClass() {
 
   static final String BINDING_NAME = "sample123";
 
+  static final String BINDING_OPERATION = "create";
```

Comment:
nit: I would use `BINDING_CREATE_OPERATION` or something more more specific name.

Reply:
I am following the same pattern as above.